src/html/index.html

```diff
@@ -10,17 +10,21 @@
         />
         <meta name="author" content="Barthélemy Paléologue" />
         <meta name="robots" content="index, follow" />
-        <title><%= htmlWebpackPlugin.options.title %></title>
+        <title>%VITE_APP_TITLE%</title>
     </head>
 
     <body>
         <canvas id="renderer"></canvas>
 
         <div id="inspectorLayer"></div>
 
-        <%= require('html-loader!./bodyEditor.html').default %> <%= require('html-loader!./spaceStationUI.html').default
-        %> <%= require('html-loader!./helmetOverlay.html').default %> <%= require('html-loader!./mainMenu.html').default
-        %> <%= require('html-loader!./pauseMenu.html').default %> <%= require('html-loader!./sidePanels.html').default
-        %>
+        <script type="module">
+            document.title = import.meta.env.VITE_APP_TITLE;
+        </script>
+
+        <script type="module" src="/src/ts/i18n.ts"></script>
+        <script type="module" src="/src/ts/index.ts"></script>
+        <script type="module" src="/src/ts/main.ts"></script>
+        <script type="module" src="/src/html/"></script>
```

Comment:
speaking about handlebars, you can probably take advantage of the templating to include the html partial files instead of using the newly created `main.ts`.

What I mean is that templating happen at compile/build time once, whereas your solution happens at runtime everytime.